passive/Find Credit Cards.js

```diff
@@ -1,22 +1,29 @@
 // CreditCard Finder by freakyclown@gmail.com
 
-function scan(ps, msg, src) {
-  // lets set up some stuff we are going to need for the alert later if we find a credit card
-  var url = msg.getRequestHeader().getURI().toString();
-  var body = msg.getResponseBody().toString();
-  var alertRisk = [0, 1, 2, 3]; //1=informational, 2=low, 3=medium, 4=high
-  var alertConfidence = [0, 1, 2, 3, 4]; //0=fp,1=low,2=medium,3=high,4=confirmed
-  var alertTitle = ["Credit Card Number(s) Disclosed (script)", ""];
-  var alertDesc = ["Credit Card number(s) was discovered.", ""];
-  var alertSolution = [
-    "why are you showing Credit and debit card numbers?",
-    "",
-  ];
-  var cweId = [0, 1];
-  var wascId = [0, 1];
+var ScanRuleMetadata = Java.type(
+  "org.zaproxy.addon.commonlib.scanrules.ScanRuleMetadata"
+);
 
-  // lets make some regular expressions for well known credit cards
-  // regex must appear within /( and )/g
```

Comment:
Should we keep this and just move it down to L27?

Reply:
Tweaked